internal/osutils/stacktrace/stacktrace.go

```diff
@@ -31,10 +34,33 @@ type Frame struct {
 var FrameCap = 20
 
 // String returns a string representation of a stacktrace
+// For example:
+//   ./package/file.go:123:file.func
+//   ./another/package.go:456:package.(*Struct).method
+//   <go>/src/runtime.s:789:runtime.func
 func (t *Stacktrace) String() string {
 	result := []string{}
 	for _, frame := range t.Frames {
-		result = append(result, fmt.Sprintf(`%s:%s:%d`, frame.Path, frame.Func, frame.Line))
+		// Shorten path from its absolute path.
+		path := frame.Path
+		if strings.HasPrefix(path, build.Default.GOROOT) {
+			// Convert "/path/to/go/distribution/file" to "<go>/file".
+			path = strings.Replace(path, build.Default.GOROOT, "<go>", 1)
+		} else if root, err := environment.GetRootPath(); err == nil {
```

Comment:
`environment.GetRootPath()` will be the same each time it is called during the same runtime. We should cache it.